ceph.spec.in

```diff
@@ -33,10 +33,12 @@
 %bcond_with selinux
 %endif
 
-# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12+ and openSUSE TW
+# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12+ (not openSUSE)
 %if 0%{?fedora} || 0%{?rhel} >= 6 || 0%{?suse_version} >= 1315
+%if ! 0%{?is_openSUSE}
```

Comment:
I doubt if the parentheses are still broken in RH/CentOS/Fedora, but if you like we can leave it separate as you suggest. How about this, then:

```
%if 0%{?fedora} || 0%{?rhel} >= 6 || 0%{?suse_version}
%if ! 0%{?is_opensuse}
```

(note the lowercase is_opensuse)